### ISSUE TITLE
[Content UX] Math text can't be copy/pasted

### ISSUE THREAD

**Issue by JRussellHuffman** [Qiskit/platypus#428]
### Steps to reproduce the problem

1. Go to https://learn.qiskit.org/course/ch-applications/solving-linear-systems-of-equations-using-hhl-and-its-qiskit-implementation (bug is on all pages)
3. Scroll down to a section with math
4. Try and copy the math text

### What is the current behavior?

Highlighting text skips over the math text
<img width="357" alt="Screen Shot 2021-10-21 at 10 19 44 AM" src="https://user-images.githubusercontent.com/7633881/138308037-11419b7c-796d-4932-b3f9-2772b09c2b46.png">

### What is the expected behavior?

To be able to copy the math text to clipboard

### Screenshots (optional but very helpful)
<img width="865" alt="Screen Shot 2021-10-21 at 10 20 33 AM" src="https://user-images.githubusercontent.com/7633881/138308195-04db360c-d85a-425a-92a4-40233f29a832.png">

### Environment (optional but very helpful)

 - Device: laptop (mac), mobile (android), tablet (ios)
 - OS: mac, android, ios
 - Browser: Safari (tablet, desktop), brave (mobile), chrome (desktop), firefox (desktop) 

### Additional context (optional)

Add any other context that could help debug the problem.

**Comment by eddybrando** [comment]
@JRussellHuffman, I think this is quite complex. Do you have an example of some other page where this is implemented? By _this_ I mean the ability to copy both regular characters and a rendered math formula to the clipboard.

**Comment by JRussellHuffman** [comment]
The example would be any page from the old version of the textbook, ([example](https://qiskit.org/textbook/ch-applications/hhl_tutorial.html)). This seems to be a known mathjax issue and there doesn't seem to be a decent way of fixing it without rolling back to an old version. 

But from a UX/user perspective, I'm still worried it's going to seem like we're removing a feature. I polled the Qiskit Advocates, and most users indicated they do at least occasionally copy/paste math from the text. 

I believe last year we brainstormed some alternatives and workarounds, but there was no solution.
